lib/tasks/create_mbrave_tags.rake

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+require 'csv'
+require 'yaml'
+
+namespace :mbrave do
+  desc 'Modifying tag groups'
+
+  #
+  # How to use it:
+  # bundle exec rake mbrave:create_tag_plates <tag_layout_template> <login> <num_plates>
+  #
+  # Example:
+  # bundle exec rake mbrave:create_tag_plates Bioscan_384_template_1_v3 admin 5
```

Comment:
I think the more likely use case for this is 'As the Lab we want a set of v3 plates for the month'. So 1 instance of each tag plate created from all 24 layout templates for v3, so 24 barcodes total. It would be risky for them to have several tag plates e.g. of layout number 4 of v3 around. And they physically can only make one tag plate at at a time from the masters. Ask Scott maybe?